Segments/Oakvael.mapproj

```diff
@@ -111398,7 +111398,6 @@
         BaseDodge = 14,
         HideDetection = 21,
         Experience = 1507,
-        BasePenetration = ShieldPenetration.VeryLight,
```

Comment:
For Very light removal, you might want to switch that to none. Otherwise it inherits from weapon.

Reply:
Just noticed Crossbow.cs would set it back to Light.

Nice catch, thx!